src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -378,24 +426,34 @@ public void setValues(JSONObject formData) {
                 "projectListFetchDelay",
                 DEFAULT_PROJECT_LIST_FETCH_DELAY);
 
-        projectListRefreshInterval = formData.optInt(
```

Comment:
`projectListRefreshInterval` and `enableProjectAutoCompletion` seems to have been removed and not replaced?

Reply:
My mistake, they are back in